dali/test/python/operator_2/test_rotate.py

```diff
@@ -234,19 +235,18 @@ def create_pipeline(backend, *args):
 
 
 def run_cases(backend1, backend2, epsilon):
-    for batch_size in [1, 4, 19]:
-        for output_size in [None, (160, 240)]:
-            for (itype, otype) in io_types:
+    for output_size in [None, (160, 240)]:
+        for (itype, otype) in io_types:
+            batch_size = np.random.choice([1, 4, 19])
```

Comment:
no need to triplicate all tests for different batch size. Like this we will hit each of them many times anyway